app/models/feed.rb

```diff
@@ -173,9 +167,7 @@ def normalize_category(cat)
   end
 
   def publish_to_apple?
-    apple_config.present? &&
-      apple_config.public_feed == self &&
-      apple_config.publish_enabled?
+    !!apple_config&.publish_to_apple?
```

Comment:
Note this method now returns true on the _private_ feed, but not on the default/public feed.